backend/app/api/endpoints/adapter/tasks.py

```
@@ -146,15 +146,25 @@ def get_group_tasks_lite(
 def get_personal_tasks_lite(
     page: int = Query(1, ge=1, description="Page number"),
     limit: int = Query(50, ge=1, le=100, description="Items per page"),
+    types: str = Query(
+        "online,offline",
+        description="Comma-separated task types to include: online (chat), offline (code), flow",
+    ),
     current_user: User = Depends(security.get_current_user),
```

Ruff (0.14.13)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)